contracts/ChainlinkPriceFeed.sol

```diff
@@ -73,7 +78,9 @@ contract ChainlinkPriceFeed is IPriceFeed, BlockContext {
                 break;
             }
 
-            weightedPrice = weightedPrice.add(currentPrice.mul(previousTimestamp.sub(currentTimestamp)));
+            timeFraction = previousTimestamp.sub(currentTimestamp);
```

Comment:
Is it possible that 2 rounds have the same timestamp?

Reply:
it's unlikely, but potentially yes. That's why we have this pr.